web/package.json

```diff
@@ -92,7 +90,6 @@
   "devDependencies": {
     "@cypress/code-coverage": "^3.10.0",
     "@cypress/instrument-cra": "^1.4.0",
-    "@testing-library/dom": "^9.0.0",
```

Comment:
testing-library/react already requires testing-library/dom 8.0. i don't think there was any reason to separately install 9.0